core/src/db/merge/MergeLayeredStrategy.cpp

```diff
@@ -14,11 +14,20 @@
 #include "db/meta/MetaConsts.h"
 #include "utils/Log.h"
 
+#include <algorithm>
 #include <map>
 #include <vector>
 
 namespace milvus {
 namespace engine {
+namespace {
+struct {
+    bool
+    operator()(meta::SegmentSchema& left, meta::SegmentSchema& right) const {
+        return left.file_size_ > right.file_size_;
+    }
+} CompareSegment;
```

Comment:
This is C-style structure ...

Reply:
struct CompareSegment {
....

Reply:
"struct CompareSegment" is easier to be searched